### PR TITLE
Decoupled docs

### PR DESCRIPTION
**Pull request by joan-ing**
Closes #

## Summary

**[Front-End Sites on Pantheon](https://docs.pantheon.io/guides/decoupled)** - A major overhaul of the Pantheon Front-End Sites documentation to create a clear set of steps that users can work through in a cohesive manner.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
| Redirect From     | To |
| ----------- | ----------- |
| docs/guides/decoupled-sites    | docs/guides/decoupled/overview       |
| docs/guides/decoupled-sites/considerations/   | docs/guides/decoupled/overview/considerations   |
|  docs/guides/decoupled-sites/site/   |  docs/guides/decoupled/overview/site    |
|  docs/guides/decoupled-sites/support/ | docs/guides/decoupled/overview/support  |
| docs/guides/decoupled-sites/faq/   | docs/guides/decoupled/overview/faq  |

**Comment by stevector** [comment]
@whitneymeredith as you're working on these changes, can you make sure we're consistent on "Front-End Sites". See https://twitter.com/chriscoyier/status/1452631050203672580

**Comment by whitneymeredith** [comment]
@stevector That's one of the things I'm focusing on!

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7880-documentation.appa.pantheon.site/guides/decoupled)

**Comment by whitneymeredith** [comment]
@stevector This is ready for review. Thanks!!

**Comment by stevector** [comment]
I'm starting a review now

File: source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Backend Only Setup
+
+You can use the default Pantheon recipe provided by Lando if you are creating a local environment for your Drupal backend only. Refer to the [Getting Started section of the Lando Pantheon Plugin documentation](https://docs.lando.dev/pantheon/getting-started.html) for more information.
+
+## Combined Frontend and Backend Setup
+
+1. Create a `.lando.yml` file in the root of the project.
+
+1. Copy the following config into `.lando.yml`. Note: Replace `%SITE-NAME%`, `%PANTHEON_SITE_ID%`, and `%PORT-NUMBER%`, with the site name, site id, and required port number for the Front-End Site.
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md:40]: @whitneymeredith have you tested these instructions? How would the developer know what to use as PORT-NUMBER?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You must manually configure edge caching and purging if you did not use the Decoupled Kit backend starter for your project.
+
+The edge caching instructions below focus on sites running on Pantheon, but can be adapted for other platforms.
+
+### Enable Caching for JSON:API
+
+1. Open your **Drupal Admin** page, select **Configuration**, select **Development**, and then select **Performance** to access the **Performance settings** page.
+
+1. Set the `Browser and proxy cache maximum age` to a value greater than zero.
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/05-cache.md:43]: Can we be more specific?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+#### Enable Edge Caching
+
+1. Run the following Composer command:
+
+  ```bash{promptUser: user}
+  composer require drupal/pantheon_advanced_page_cache
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/05-cache.md:50]: Shouldn't this module already be in the starter?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  ```
+
+1. Enable the Pantheon Advanced Page Cache module.
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/05-cache.md:53]: And I'd expect the starter would enable it too.

File: source/content/guides/decoupled/drupal-backend-starters/06-configure-existing.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+- Update your starter kit to use the default image field instead of the Media
+  field.
+
+### Ensure That Your Content Uses URL Aliases
+
+The starter kit uses the Decoupled Router module to determine the path where your Drupal content should display. As a result, while the pattern used to define your paths does not matter, your page and article content must have a path alias of some kind. This can be defined manually during content creation, or automatically using the [Pathauto module](https://www.drupal.org/project/pathauto).
+
+### Create Supporting Content
+
+The Front-End Sites starter kits assume that there is at least one published article and page in your Drupal backend. If your site does not have any article or page content, you should create some before proceeding.
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/06-configure-existing.md:61]: Is this a roundabout way of saying "the starterkits don't fail gracefully if there isn't content" ?

File: source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides information on how to configure your cross-origin resource sharing (CORS).
```
Comment (stevector) [inline source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md:19]: This page is nearly identical to https://decoupledkit.pantheon.io/docs/backend-starters/decoupled-drupal/cors-configuration

What's the long term plan for managing similar/identical docs across these two sites?

**Review by stevector — CHANGES_REQUESTED** [review]
Hi @whitneymeredith, I haven't reviewed everything in-depth. But I figured I'd submit what I have so far. I think my main question is: What's the long-term plan for managing documentation between here and https://decoupledkit.pantheon.io/ what information should go where?

**Comment by whitneymeredith** [comment]
> Hi @whitneymeredith, I haven't reviewed everything in-depth. But I figured I'd submit what I have so far. I think my main question is: What's the long-term plan for managing documentation between here and https://decoupledkit.pantheon.io/ what information should go where?
@stevector All public documentation (not including repo and packages info) should be on the Docs site.

File: source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md (hunk 6 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md:19]: @stevector All Front-End Site public content will be on the Docs site (expect for packages info and repo info). Over time the decoupledkit. pantheon.io will be phased out.

File: source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md (hunk 1 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md:40]: @stevector All technical steps and contextual content was taken from the decoupledkit. pantheon.io site vetted and maintained by Brian and Coby.

Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md:40]: @CobyPear @backlineint How does the user know what port number to use?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md (hunk 2 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:43]: @backlineint @CobyPear can "value greater than zero" be changed to "to the length of time you want this information cached." ? I'd like to add an example, such as 1 hour. What is the format the value takes? Would an hour be set to '60'?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md (hunk 3 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:50]: @CobyPear @backlineint Can you clarify this step?

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md (hunk 4 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:53]: @CobyPear @backlineint We could use clarification here as well.

File: source/content/guides/decoupled/overview/01-introduction.md [hunk 7]
```diff
@@ -40,7 +38,7 @@ Decoupled Architecture is a site architecture that combines the speed and agilit
 
 Pantheon Front-End Sites provide users with tools that improve the experience of building a decoupled frontend that sources data from a CMS backend.
 
-With decoupled sites, a CMS site can be linked to a single decoupled site, multiple sites, or the user does not need a CMS at all and can link directly to a frontend application.
+With decoupled sites, a CMS site can be linked to a single decoupled site, multiple sites, or you can not use a CMS at all and link directly to a frontend application.
```
[the remark this answers is not in the document]
Comment (stevector) [reply source/content/guides/decoupled/overview/01-introduction.md:41]: when used as a noun, it's `back end` or `front end`. But it's almost always clearer to add a more specific noun. So instead of `Drupal is the back end` it's clearer to write `Drupal is the back-end CMS`. and `Drupal is the backend` is just incorrect.

File: source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md (hunk 6 above)
Comment (stevector) [reply source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md:19]: @EdwardAngert do you have perspective here? Do you know what Joan intended?

File: source/content/guides/decoupled/overview/01-introduction.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled/overview/01-introduction.md:41]: @stevector I think the tweet is referencing the Microsoft Style guide. Backend an frontend are also considered acceptable, and allows us to avoid the hyphen to create greater visual distinction between the product name and the use of the general technical term (Front-End Sites and frontend site).

File: source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md (hunk 6 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md:19]: @backlineint I think both sites will continue to exist for the duration of EA. However, I was under the impression that when the product eventually reaches GA, that the decoupledkit. pantheon.io site would most likely be phased out. This might be a misunderstanding on my part. The Docs site content is for developer use. The intention is to provide a conceptual introduction to the product and the technical information and steps required to use the product. This is the same format that all our guides use.

File: source/content/guides/decoupled/drupal-backend-starters/09-build-hooks.md
Comment (CobyPear) [inline source/content/guides/decoupled/drupal-backend-starters/09-build-hooks.md:1]: I see how this (and the WP counterpart) is linked from the 'Build Hooks' page which makes sense, but since the backend config could be used for any build hook, I would like to see this info stay on the dev docs if possible.

File: source/content/guides/decoupled/drupal-nextjs-frontend-starters/03-create.md
Comment (CobyPear) [inline source/content/guides/decoupled/drupal-nextjs-frontend-starters/03-create.md:1]: IMO, the platform docs should only be concerned with how to create a starter-kit from the dashboard.

File: source/content/guides/decoupled/drupal-nextjs-frontend-starters/04-environment-variables.md
Comment (CobyPear) [inline source/content/guides/decoupled/drupal-nextjs-frontend-starters/04-environment-variables.md:1]: This one is mostly concerned with local development. I could see a more generic guide (if we don't already have one) about setting environment variables in the Dashboard for a Front-End Site, but the instructions in the doc are only relevant locally.

File: source/content/guides/decoupled/drupal-nextjs-frontend-starters/07-style.md
Comment (CobyPear) [inline source/content/guides/decoupled/drupal-nextjs-frontend-starters/07-style.md:1]: Similar to 'your first customization', I could see this living on the platform docs, however, we actually have something in our new(ish) command line interface that can add tailwindcss for you, which is mostly all this doc is explaining. So I wonder if this should stay on the dev docs for now and we can revisit it when the CLI is more of a first class citizen (we are working on that now).

File: source/content/guides/decoupled/no-starter-kit/02-create.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+* The repository should not be empty.
+
+* The repository has a `package.json` file.
+
+* The repository has only **one** lock file.
+
+    <Alert title="Note"  type="info" >
+
+    You cannot have a `package-lock.json` and `yarn.lock` file. You will receive an error message if you try to import a repository with both a `package-lock.json` and `yarn.lock`.
```
Comment (CobyPear) [inline source/content/guides/decoupled/no-starter-kit/02-create.md:38]: We support `pnpm` as well; should this note include `pnpm-lock.yaml`?

File: source/content/guides/decoupled/no-starter-kit/02-create.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Optional. Link your CMS.
+
+    <Alert title="Note"  type="info" >
+
+    You are not required to have a CMS when using this method. If you are using a specified CMS, you can link your CMS backend and the site environment from which to source content. However, this is more applicable when using the Decoupled starter kit templates.
+
+    </Alert>
+
+1. Ensure **Make this a private repository** is selected.
```
Comment (CobyPear) [inline source/content/guides/decoupled/no-starter-kit/02-create.md:67]: I don't know if this step is still necessary. I'm in favor of removing it.

File: source/content/guides/decoupled/no-starter-kit/02-create.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Optional. Add configurations and then click **Continue**.
+
+    * You can add specifications to help manage your root directory and build settings.
+    * You can set the Environment Variables in the Advanced Setting section.
+
+    <Alert title="Note"  type="info" >
+
+     Basic builds will function without setting environment variables if the CMS site was selected during decoupled site creation. Environment variables are not necessary for optional features such as Decoupled Preview.
```
Comment (CobyPear) [inline source/content/guides/decoupled/no-starter-kit/02-create.md:76]: > Basic builds will function without setting environment variables if the CMS site was selected during decoupled site creation.

This is true for the starter kit flow, but in the case of a user importing a repo, we can not guaruntee anything about the build step since we have no idea what env vars are or are not required for their build.

Also I think this `not` is extraneous:

```suggestion
     Basic builds will function without setting environment variables if the CMS site was selected during decoupled site creation. Environment variables are necessary for some optional features such as Decoupled Preview.
```

File: source/content/guides/decoupled/no-starter-kit/02-create.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+     </Alert>
+
+    - You are directed to a new page with the site name, deployment, and build information/ status. You will receive a message that your decoupled site has successfully been added to Pantheon.
+
+1. Click **Build Details** to view the build log.
```
Comment (CobyPear) [inline source/content/guides/decoupled/no-starter-kit/02-create.md:82]: Here is a place I could see having a screenshot, however, this section of the dashboard is still in flux enough that it gives me some pause.

File: source/content/guides/decoupled/overview/04-site.md [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+
+This section provides configuration information for Front-End Sites.
+
+## Prerequisites
+
+* The **Decoupled** tab is enabled in your dashboard after onboarding.
+
+* Your site repository is in GitHub.
+
+* You have a domain to use for the Front-End site.
```
Comment (CobyPear) [inline source/content/guides/decoupled/overview/04-site.md:29]: This should not be necessary to create a Front-End Site

File: source/content/guides/decoupled/overview/04-site.md [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+
+* You will use Next.js or Gatsby as the frontend framework.
```
Comment (CobyPear) [inline source/content/guides/decoupled/overview/04-site.md:31]: Is there a way we can signal that any JavaScript based runtime, or any static site generator that outputs html/css/js is useable, but we only fully support Next.js and Gatsby?

File: source/content/guides/decoupled/overview/04-site.md [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+
+* (Optional). If using a pre-configured starter kit template, you will use one of the following combinations:
+
+    * [Drupal and Next.js](/guides/decoupled/drupal-nextjs-frontend-starters/)
+
+    * [WordPress and Next.js](/guides/decoupled/wp-nextjs-frontend-starters/)
+
+    * [WordPress and Gatsby](/guides/decoupled/wp-gatsby-frontend-starters/)
+
+    > **NOTE**: Other starter kit configurations are not currently supported.
```
Comment (CobyPear) [inline source/content/guides/decoupled/overview/04-site.md:43]: Same here. Is it possible to show that other configs are _possible_, however they may not be fully _supported_?

File: source/content/guides/decoupled/overview/04-site.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+
+
+### Compatibility
+
+You can create an application from scratch by importing a repository or you can use the Pantheon starter kit templates to develop a Front-End Site.
+
+You can also create a site without a CMS. This is done by importing the repository and using Next.js or Gatsby as the static site generator.
```
Comment (CobyPear) [inline source/content/guides/decoupled/overview/04-site.md:50]: This is really just a nitpick so feel free to disregard :)

```suggestion
You can also create a site without a CMS. This is done by utilizing the import repository feature during site creation.
```

I don't love the faux constraint of 'Next.js or Gatsby'. I know there is some future work to be done to make the drop down menu more accurate as to the type of rendering, not the framework (at the moment next = ssr, gatsby = ssg). I'm wondering if changing this message here a bit can clear up some ambiguity? Or will this cause more confusion?

File: source/content/guides/decoupled/overview/04-site.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+
+### Frontend Starters
+
+- [Drupal + Next.js Frontend Starter](/guides/decoupled/drupal-nextjs-frontend-starters)
+- [WordPress + Next.js Frontend Starter](/guides/decoupled/wp-nextjs-frontend-starters)
+- [WordPress + Gatsby Frontend Starter](/guides/decoupled/wp-gatsby-frontend-starters)
+
+### No Starter
+
+- [Create a Front-End Site without a Starter](/guides/decoupled/no-starter-kit)
```
Comment (CobyPear) [inline source/content/guides/decoupled/overview/04-site.md:78]: I could see this section as its own page, and instead of linking to the guides, putting the guides in a tabbed section.

**Review by CobyPear — CHANGES_REQUESTED** [review]
As per our discussion, I have marked some of the guides that I think can stay on `decoupledkit.pantheon.io`, but there are a few I am a bit unsure of.

Please note: I only made comments on the Drupal BE starter but same comments apply to the WP analogs. Same with the FE starter next+drupal – those comments also apply to the next+wp and gatsby+wp sections.

File: source/content/guides/decoupled/drupal-backend-starters/01-introduction.md
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/01-introduction.md:1]: @CobyPear Thanks! The content is strong and helps inform and orientate users on how to use the feature. Can you help remove open-source specific steps from the content?

File: source/partials/decoupled-create-in-dashboard.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+
+Follow the steps below to access the Front-End Site decoupled architecture and start developing the frontend application in Pantheon.
+
+1. Log in to your **Pantheon Dashboard** and select the **Sites** page.
+
+1. Click the **Decoupled** tab and then click **+Create New Site**.
+
+    <Alert title="Note"  type="info" >
+
+    You cannot use a . (period) or _ (underscore) for site and Multidev names.
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/partials/decoupled-create-in-dashboard.md:23]: Added this content to Overview/Considerations

File: source/content/guides/decoupled/overview/04-site.md (hunk 15 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/overview/04-site.md:50]: Committed!

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:1]: updated!

File: source/content/guides/decoupled/overview/04-site.md (hunk 13 above)
Comment (whitneymeredith) [reply source/content/guides/decoupled/overview/04-site.md:31]: Added!

File: source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md [hunk 18]
```diff
@@ -72,29 +35,30 @@ Make sure you meet the prerequisites below before you continue.
...
+    </Alert>
+
 1. Click the **Gatsby + WordPress** template, select your Git provider under **Choose your Git provider**, and then click **Continue**.
 
+  ![select a starter](../../../../images/decoupled-select-starter.png)
+
 1. Select your desired GitHub account and enter the repository name. You can select **Make this a private repository** to provide greater security for your Git repository.
 
-1. Select the WordPress CMS backend from the drop-down menu.
+1. Select the desired WordPress CMS backend from the **Link Your CMS** drop-down menu.
```
Comment (CobyPear) [inline source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md:54]: Should there be another optional step here to point out the Environment Variables section in the "Advanced Settings" Tab during site creation?

Comment (whitneymeredith) [reply source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md:54]: Added! Thank you!

File: source/content/guides/decoupled/drupal-backend-starters/05-cache.md (hunk 3 above)
Comment (stevector) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:50]: I'm skeptical that "as much information as possible to use Front-End Sites" is the right guiding light here. I think it's more "the right amount of the right information to use Front-End Sites." For someone using our Front-End Sites product, we should not be telling them to run `composer require drupal/pantheon_advanced_page_cache` because that module is already present if they're using the starter provided by Front-End Sites.

I agree with you @whitneymeredith that repeatedly linking to another site in the flow of a single page or series of pages is jarring and adds friction. But this is info that doesn't need to be linked at all (or maybe just linked at the end).

Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/05-cache.md:50]: @stevector The introductory text at the top states: "Cache settings are configured out-of-the-box for you, but there are circumstances in which you might need to make your own configurations, for example, your CMS is not on Pantheon but your frontend is on Pantheon." We're not telling users this is a required a step.

File: source/content/guides/decoupled/overview/04-site.md [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+| WordPress  |   5.9 or higher |
+
+## Before You Use Pantheon Starter Kits
+
+The following components are required to create and configure a new decoupled project using a Front-End Site starter kit:
+
+1. Install [Composer](https://getcomposer.org/download/) globally.
+    * Composer is required for the CMS backends.
+
+1. Install [Terminus](/terminus/install) version 3.0.0 or higher.
```
Comment (backlineint) [inline source/content/guides/decoupled/overview/04-site.md:77]: @whitneymeredith using Terminus and also the following three items about tokens are optional. This is presented as required.

Comment (whitneymeredith) [reply source/content/guides/decoupled/overview/04-site.md:77]: Updated! Thanks!!

File: source/content/guides/decoupled/drupal-nextjs-frontend-starters/04-preview.md [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+site.
+
+1. Set the URL to point to `http(s)://{YOUR_SITE_URL}/api/preview`, replacing
+`{YOUR_SITE_URL}` with the URL of your Front-End Site, or `localhost:3000` to test the preview locally.
+
+1. Set a secret for the Preview Site and note this value down.
+
+Now you have all of the credentials needed to make authenticated requests to the Drupal instance, including the ability to preview content.
+
+## Use `getPreview`
```
Comment (backlineint) [inline source/content/guides/decoupled/drupal-nextjs-frontend-starters/04-preview.md:45]: I'm not convinced that this section belongs here at all. If it was included, there should be a section prior to this that explains that preview for articles and pages will work without configuration if you use our starter projects. The examples below would only be necessary if they were customizing or implementing preview for other content types.

File: source/content/guides/decoupled/drupal-nextjs-frontend-starters/05-customization.md
Comment (backlineint) [inline source/content/guides/decoupled/drupal-nextjs-frontend-starters/05-customization.md:1]: I don't think this entry belongs in the platform docs. This would be the equivalent of having a section in our Drupal docs providing code samples for creating custom modules or customizing a subtheme. If that is out of scope for CMS sites, why would it be in scope here?

**Review by backlineint — CHANGES_REQUESTED** [review]
This is a partial review - but I wanted to share sooner rather than later.

I've focused on overview and Next Drupal starter so far. I'll follow up with comments on the other starters.

It's a lot to go though - sorry if any of it comes off as blunt. I know we're in a tough spot and there is a lot to revise here. I just feel strongly that in the current state these docs make Front-end Sites come off as unnecessarily complex.

File: source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md (hunk 1 above)
Comment (backlineint) [reply source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md:40]: The port number will vary based on the JS framework you are using. Next uses 3000. Gatsby uses 8000. They can be configured to use other ports. Other frameworks may use other ports.

I wasn't expecting these instructions to be ported to the platform docs. To be honest, I don't think they are mature enough for that. I think we would need a better solution than copying and pasting a large config file like this.

File: source/content/guides/decoupled/drupal-backend-starters/06-configure-existing.md
[the remark this answers is not in the document]
Comment (backlineint) [reply source/content/guides/decoupled/drupal-backend-starters/06-configure-existing.md:1]: @whitneymeredith I'm not sure why @CobyPear 's comment was marked as resolved here. I also not sure that this document belongs in the platform docs, but if it does, it can't be in the `Decoupled Drupal Backend Starter for Front-End Sites` section. This entry only applies to users who choose not to use the Drupal Backend Starter for Front End Sites, and would likely be very confusing to users who do.

File: source/content/guides/decoupled/wp-backend-starters/01-introduction.md [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+
+Front-End Sites on Pantheon allow you to use [decoupled architecture](/guides/decoupled/overview/#what-is-a-decoupled-site) to separate your frontend and backend into distinct entities.
+
+You can use the decoupled WordPress backend starter kit to streamline the creation of your Front-End Site on Pantheon. This guide shows you how to install, set up local development, manage WordPress dependencies, configure build hooks, and more.
+
+## Requirements and Considerations
+
+Review [requirements and considerations](/guides/decoupled/overview/considerations) for Front-End Sites on Pantheon before you begin your project.
+
+## WordPress Kit Packages
```
Comment (backlineint) [inline source/content/guides/decoupled/wp-backend-starters/01-introduction.md:28]: My comments from the related Drupal section apply here as well.

File: source/content/guides/decoupled/wp-backend-starters/03-local-dev.md
Comment (backlineint) [inline source/content/guides/decoupled/wp-backend-starters/03-local-dev.md:1]: My comments for the Drupal version of this entry apply here as well.

File: source/content/guides/decoupled/wp-backend-starters/04-configure-existing.md
Comment (backlineint) [inline source/content/guides/decoupled/wp-backend-starters/04-configure-existing.md:1]: My comments for the Drupal version of this entry apply here as well.

**Review by backlineint — CHANGES_REQUESTED** [review]
Finished going through the other entries. 

There are still a lot of entries taken mostly wholesale from the developer docs that I wouldn't have expected to be part of the platform docs (especially if the goal for this PR was mainly to restructure the the docs.) If we believe they need to be included here, some need major revisions to make sense in the context of the platform docs.

File: source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+
+## Create Your Project on Pantheon
+
+Make sure you meet the prerequisites below before you continue.
+
+* You are using a Git repository and you have connected your GitHub account.
+
+* You are using WordPress as your CMS.
+
+* You have installed your WordPress site using the [Decoupled WordPress Recommended Project](https://github.com/pantheon-upstreams/decoupled-wordpress-composer-managed) configuration, which has the `wp-graphql` and `wp-gatsby` plugins enabled.
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md:36]: updated

File: source/content/guides/decoupled/drupal-backend-starters/02-create.md [hunk 23]
```diff
@@ -0,0 +1,221 @@
...
+
+1. Name your site, select your decoupled environment from the **Choose a Workspace for the Site** drop-down menu, and then click **Continue**.
+
+1. Confirm your organization selection when prompted in the **Confirm Organization Selection** dialogue box. The Deploying Decoupled Drupal Composer Managed progress indicator displays while your site is spinning up.
+
+1. Click **Visit your Pantheon Dashboard** when the site spin up completes.
+
+1. Click the **Visit Development Site** button to install Drupal.
+
+1. Select either the `Pantheon Decoupled Profile`, or the
```
[the remark this answers is not in the document]
Comment (whitneymeredith) [reply source/content/guides/decoupled/drupal-backend-starters/02-create.md:58]: added

**Comment by EdwardAngert** [comment]
<https://pr-7880-documentation.appa.pantheon.site/guides/decoupled>

**Comment by whitneymeredith** [comment]
@backlineint  Thanks so much for clarifying the quality of the documentation on the Decoupled Kit site.  My initial assessment was that it was great documentation that would be awesome to have on the Docs site. It's a good to know I wasn't wrong about it being strong content :)

I agree with you that Content in a readme file is official documentation.  In my comment, I simply meant that the Decoupled Kit site looks very similar to the [docs.pantheon.io](http://docs.pantheon.io/) site, which is the official documentation for the Pantheon platform. I didn’t mean to imply that your content wasn't official for the open source Decoupled Kit project. 

I'm glad  we agree on improving the branding to distinguish between the sites better. This is something that I think will be useful for other open source projects as well. Let me know how I can help with this!

**Comment by EdwardAngert** [comment]
@whitneymeredith is it okay to close this one?

**Comment by whitneymeredith** [comment]
> @whitneymeredith is it okay to close this one?

Yes!
